bl/evaluator_test.go

```diff
@@ -252,7 +252,7 @@ func TestPrintResults(t *testing.T) {
 		}{},
 	}
 
-	evaluator.PrintResults(&results, "cli_id")
+	evaluator.PrintResults(&results, "cli_id", "")
```

Comment:
I think we should add tests to check that the other formats output works as expected

Reply:
added tests